lib/airbrake-ruby/nested_exception.rb

```diff
@@ -35,7 +35,7 @@ def unwind_exceptions
     end
 
     def parse_backtrace(exception)
-      return if exception.backtrace.nil? || exception.backtrace.none?
+      return [] if exception.backtrace.nil? || exception.backtrace.none?
       Backtrace.parse(exception)
```

Comment:
Moved the check to `Backtrace.parse`. Wanted to do the latter, but the Java exceptions check in `Backtrace` makes it a more complex refactoring (which is not worth it).